### PR TITLE
Added pytest unit testing

### PR DESCRIPTION
**Pull request by nereasastre**
## 🔗 Tasks
- [Task 1](trello.url)
- [Task 2](trello.url)

## 👉 What?
Added automated tests with pytest. Updated requirements.txt to include pytest

## 🤷 Why?
Automated tests help us check many different files in just one run. With them we can see if the pitch extractors work

## ⚙️ How?
You can run the automated tests by installing pytest and running python -m pytest in the command line

## 👁‍🗨 Testing?
The tests cover the extraction of real and synthetic signals with an error threshold of 2
## 📸 Screenshots (optional)
Failed tests correspond to real-life sounds: right now our extractors have an error > 2 Hz
image

### CODE REVIEW

**Comment by nereasastre** [comment]
he visto que en el requirements.txt he quitado cosas sin querer, puede ser @APPIAHTECH ? yo quería añadir pytest

File: requirements.txt [hunk 1]
```diff
@@ -1,15 +1,11 @@
-asgiref==3.4.1
```
Comment (juliatogr) [inline requirements.txt:1]: Porque has modificado el requirements.txt? hay que ir con cuidado con eso. Creo que tienes una versión antigua.

**Review by APPIAHTECH — APPROVED** [review]
Buena!

Tema requirement.txt el orden cambio, pero esta todo.

**Review by juliatogr — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (nereasastre) [reply requirements.txt:1]: lo modifiqué porque necesitaba añadir pytest para los tests

**Pull request merged**